jbpm-task-assigning/kie-server-api-task-assigning/src/main/java/org/kie/server/api/model/taskassigning/QueryParamName.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.server.api.model.taskassigning;
```

Comment:
This splitting is because right now I have this development in a completely separate structure and with the aim of adding into the kie-server-parent structure the "less as possible". This was good for keeping this development isolated as much as possible. And there are a few classes there.
BUT, the final destination of all this development is kie-server-parent module, so when this PR is approved next step is to move all this classes to the correspnding kie-server-api.
So consider this module as "temporal".